src/test/java/edu/ucla/library/iiif/fester/verticles/FakeS3BucketVerticle.java

```diff
@@ -37,13 +36,14 @@
     private File myTmpDir;
 
     @Override
-    public void start(final Promise<Void> aPromise) throws Exception {
-        final String deploymentID = deploymentID();
+    public void start() throws Exception {
+        super.start();
 
         LOGGER.debug(MessageCodes.MFS_110, getClass().getName(), deploymentID());
 
         if (myTmpDir == null) {
-            myTmpDir = Files.createTempDirectory(deploymentID).toFile();
+            // Creates a tmp dir from the deploymentID, adding additional random numbers after the underscore
+            myTmpDir = Files.createTempDirectory(deploymentID() + "_").toFile();
```

Comment:
Why this change?

Reply:
(the `_`)

Reply:
The directory name is comprised of the prefix that I pass in (the deployment ID) and a random number. The underscore makes a clear delimiter that I can use to regexp match the first part of the dir name (the deployment ID) to find the directory.

Reply:
That makes sense.